types/readable-stream/index.d.ts

```diff
@@ -91,18 +91,14 @@ declare namespace _Readable {
         on(ev: string | symbol, fn: (...args: any[]) => void): this;
 
         _undestroy(): void;
-        [Symbol.asyncIterator](): AsyncIterableIterator<string | Buffer>;
+        [Symbol.asyncIterator](): AsyncIterableIterator<any>;
         // end-Readable
 
         constructor(options?: DuplexOptions);
     }
 
     // ==== _stream_passthrough ====
-    class PassThrough extends Transform implements stream.PassThrough {
-        constructor(options?: TransformOptions);
-
-        _transform<T>(chunk: T, encoding: string | null | undefined, callback: (error: any, data: T) => void): void;
-    }
+    class PassThrough extends stream.PassThrough {}
```

Comment:
I believe the purpose of the PassThrough override is to allow users to get stronger typing in their callbacks via the generic `T` used by both the `chunk` and `callback(..., 'data')` parameter.

Reply:
The reason of this PR is because this override breaks any change in `@types/node` stream module.

I'm trying to pass another PR, but because of this override it breaks, who will have a really stronger typing. Then we could extend it from here.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37354#issuecomment-519326105

Reply:
The reason of this PR is because this override breaks any change in `@types/node` stream module.

I'm trying to pass another PR, but because of this override it breaks, who will have a really stronger typing. Then we could extend it from here.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37354#issuecomment-519326105

Reply:
Forgot it, in fact It was a bug on node type definition side, I did fixed that on the other PR, sorry for wasting your time.